src/main/kotlin/com/terraformation/backend/accelerator/event/ModuleEventStartingEvent.kt

```diff
@@ -0,0 +1,6 @@
+package com.terraformation.backend.accelerator.event
+
+import com.terraformation.backend.db.accelerator.EventId
+
+/** Published when an event is starting. Ignored if event has been rescheduled. */
```

Comment:
The second sentence here doesn't seem accurate to me -- doesn't the rescheduling check happen before this event gets published?